.tool-versions

```diff
@@ -1,2 +1,2 @@
-elixir 1.13.0
```

Comment:
The intention here is to support versions as early as possible, to keep the library compatible with as many versions of Elixir and OTP as possible. I'll add a comment to that effect.